### PR TITLE
added add_script_to_menu method

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -1605,7 +1604,34 @@ def install(package_path, package_name=None):
...
+    @pyaedt_function_handler()
+    def add_script_to_menu(self, toolkit_name, script_path, script_image=None, product="Project"):  # pragma: no cover
+        """Add a script to Ribbon menu. This functionality is available from AEDT 2023 R2.
+        PyAEDT must be installed in AEDT to allow this functionality to run.
+        Please refer to ``"https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html"``.
+
+        Parameters
+        ----------
+        toolkit_name : str
+            Name of the toolkit as it will appear in AEDT.
```
Comment (PipKat) [inline pyaedt/desktop.py:1620]:
```suggestion
            Name of the toolkit to appear in AEDT.
```

File: pyaedt/desktop.py [hunk 2]
```diff
@@ -1605,7 +1604,34 @@ def install(package_path, package_name=None):
...
+        script_path : str
+            Full path to the script file.
+        script_image : str, optional
+            Full path to the image logo (30x30 pixels png) to be added into the UI.
```
Comment (PipKat) [inline pyaedt/desktop.py:1624]:
```suggestion
            Full path to the image logo (a 30x30 pixel PNG file) to add to the UI.
```

File: pyaedt/desktop.py [hunk 3]
```diff
@@ -1605,7 +1604,34 @@ def install(package_path, package_name=None):
...
+        product : str, optional
+            Product to which the toolkit will apply. It can be Project to apply to all designs or specific (eg. HFSS).
```
Comment (PipKat) [inline pyaedt/desktop.py:1626]:
```suggestion
            Product to which the toolkit applies. The default is ``"Project"``, in which case it
            applies to all designs. You can also specify a product, such as ``"HFSS"``.
```
Would you use lowercase for ``"HFSS"``.

**Review by MaxJPRey — APPROVED** [review]
Nice to have it available from PyAEDT.

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**